graphql/resolvers.go

```diff
@@ -1346,6 +1346,11 @@ func (r *queryResolver) TaskLogs(ctx context.Context, taskID string) (*RecentTas
 		return nil, ResourceNotFound.Send(ctx, fmt.Sprintf("could not find project '%s'", t.Project))
 	}
 
+	var taskExecution int
+	taskExecution = t.Execution
```

Comment:
The logic here is kind of flawed, because t.Execution will always be the latest one. Since you changed the schema to make it non-nullable, I would just assume that execution is not null and use it everywhere

Reply:
oh wait you made it non-nullable for taskTests but not taskLogs. Can it be non-nullable for both?

Reply:
I figured if the query doesn't provide an execution it should always return the latest  execution. But you are right its probably better to make it always required.

Reply:
Hey it would but we can also just time the deploys to go out at the same time.

Reply:
No matter how you time it, there can always be someone with the old version of spruce open so they'll hit an error

Reply:
Just pushed up to make the fields nullable again